packages/lumx-react/src/components/thumbnail/Thumbnail.tsx

```diff
@@ -144,13 +146,13 @@ const Thumbnail: React.FC<ThumbnailProps> = ({
             {...props}
         >
             {aspectRatio === AspectRatio.original ? (
-                <img className={`${CLASSNAME}__image`} src={image} alt={alt} loading={loading} />
+                <img ref={focusImageRef} className={`${CLASSNAME}__image`} src={image} alt={alt} loading={loading} />
             ) : (
                 <div className={`${CLASSNAME}__background`}>
                     <img
                         ref={focusImageRef}
                         className={`${CLASSNAME}__focused-image`}
-                        crossOrigin="anonymous"
+                        crossOrigin={isInternetExplorer ? undefined : 'anonymous'}
```

Comment:
`isInternetExplorer` is a function, right?
```suggestion
                        crossOrigin={isInternetExplorer() ? undefined : 'anonymous'}
```